constructor/osx/run_installation.sh

```diff
@@ -60,20 +71,27 @@ for env_pkgs in "${PREFIX}"/pkgs/envs/*/; do
     else
         env_channels="__CHANNELS__"
     fi
-    # TODO: custom channels per env?
+    if [[ -f "$PREFIX/pkgs/user_wants_shortcuts" ]]; then
+        # This file is guaranteed to exist, even if empty
+        env_shortcuts=$(cat "${env_pkgs}shortcuts.txt")
+        rm -f "${env_pkgs}shortcuts.txt"
+    else
+        env_shortcuts="--no-shortcuts"
+    fi
     # TODO: custom shortcuts per env?
+    # shellcheck disable=SC2086
```

Comment:
hm... disbaling SC2086 prevents detection of missing quotes for the entire command... 

does the command rely on splitting the `$env_shortcuts` parameter on spaces... there are examples in https://www.shellcheck.net/wiki/SC2086 on how to do that without entirely disabling SC2086.

Reply:
Yes, it has to split. I tried the array method but that's not POSIX, and then I gave up. 😂